cmd/rbac/rbac_revoke.go

```diff
@@ -49,7 +49,7 @@ func RunRevokePermissionsCommand(args *RevokePermissionsArgs) di.CmdWithApiFn {
 			return common.NewResponseError(revokeErr, revokeResp)
 		}
 
-		description, descrResp, descrErr := describePermissions(cli, api, args.Subject, args.Permission, args.Resource).Execute()
+		description, descrResp, descrErr := describePermissions(cli, api, args.Subject, args.Permission, "").Execute()
```

Comment:
will this drop the `permissions` parameter, or set it to the empty string?  (I wonder how this works on the stackstate side)

Reply:
it drops the resource parameter which is optional and then the effect is you list/describe all the remaining grants for that permission to the user